GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -623,6 +623,7 @@ private void writeAnswer(
             boolean useQuestionId) {
 
         assert value != null;
+        value = value.trim();
```

Comment:
This may not be necessary since we trim the value at `BulkDataServiceClient.parseSurveyInstanceResponse`